README.md

```diff
@@ -29,7 +29,7 @@ get all the news about botogram in its [Telegram channel][channel].
 > Please note botogram currently doesn't support some of the upstream API
 > features. All of them will be implemented in botogram 1.0
 
-**Supported Python versions**: 3.4, 3.5  
```

Comment:
That was intended.

In Markdown, [you need two trailing whitespaces](http://daringfireball.net/projects/markdown/syntax#p) after a line in order to put an HTML `<br>`. Apparently GitHub doesn't need those, but I'd like to have this readme as platform-agnostic as possible.